website/src/pages/privacy-policy.tsx

```diff
@@ -0,0 +1,414 @@
+            reasons of public interest. The right of objection can be exercised at any time by contacting the respective
+            controller.
+          </Container>
+
+          <Heading as="h3" size="lg">
+            7.9 Automated individual decision-making, including profiling
+          </Heading>
+
+          <Container>
+            You have the right not to be subject to a decision based solely on automated processing, including
+            profiling, which produces legal effects for you or similarly significantly affects you. This does not apply
+            if the decision: is necessary for entering into, or performance of, a contract between the data subject and
+            a data controller, is authorized by Union or Member State law to which the controller is subject and which
+            also lays down suitable measures to safeguard the data subject’s rights and freedoms and legitimate
+            interests, or is based on the data subject’s explicit consent. The controller shall implement suitable
+            measures to safeguard the data subject’s rights and freedoms and legitimate interests, at least the right to
+            obtain human intervention on the part of the controller, to express his or her point of view and to contest
+            the decision. This right can be exercised by the data subject at any time by contacting the respective
+            controller.
+          </Container>
+
+          <Heading as="h3" size="lg">
+            7.10 Right to lodge a complaint with a supervisory authority
+          </Heading>
+
+          <Container>
+            You also have the right, without prejudice to any other administrative or judicial remedy, to lodge a
+            complaint with a supervisory authority, in particular in the Member State of your habitual residence, place
+            of work or place of the alleged infringement if you as data subject consider that the processing of personal
+            data relating to you infringes this Regulation.
+          </Container>
+
+          <Heading as="h3" size="lg">
+            7.11 Right to effective judicial remedy
+          </Heading>
+
+          <Container>
+            Without prejudice to any other available administrative or judicial remedy, including the right to lodge a
+            complaint with a supervisory authority pursuant to Article 77 GDPR, you have the right to an effective
+            judicial remedy if you consider that your rights under this Regulation have been infringed as a result of
+            the processing of your personal data in breach of this Regulation.
+          </Container>
+
+          <Heading>Submitting requests</Heading>
+          <Container>
+            Email <a href="mailto:privacy@open-assistant.io">privacy@open-assistant.io</a>
+          </Container>
+        </Container>
+      </main>
+    </>
+  );
+};
+
+PrivacyPolicy.getLayout = (page) => (
```

Comment:
makes since to extract the layout with transparent header to a component? it is used 2 times here and one for the homepage.

Reply:
Done